ros_buildfarm/templates/misc/rosdistro_cache_job.xml.em

```diff
@@ -79,6 +79,9 @@
         ' --cidfile=$WORKSPACE/docker_generate_rosdistro_cache/docker.cid' +
         ' --net=host' +
         ' -v $WORKSPACE/rosdistro_cache:/tmp/rosdistro_cache' +
+        ' -v $HOME/.ssh/known_hosts:/etc/ssh/ssh_known_hosts:ro' +
+        ' -v $SSH_AUTH_SOCK:/tmp/ssh_auth_sock' +
+        ' -e SSH_AUTH_SOCK=/tmp/ssh_auth_sock' +
```

Comment:
If this is only necessary when `git_ssh_credential_id` is set I think it should also be only passed conditionally. Because it keeps the simple case simple and implicitly documents how these arguments are being used.